sensor_msgs/Image fullImage
---
sensor_msgs/Image lRectified
sensor_msgs/Image rRectified
float64 debayerTime
float64 rectificationTime
float64 intRectificationTime
